scripts/init.sql

```diff
@@ -8,14 +14,14 @@ CREATE TABLE "agencies" (
     "address_line_2" varchar(255),
     "city" varchar(100) NOT NULL,
     "state" varchar(100) NOT NULL,
-    "country" varchar(100) NOT NULL,
+    "country_code" varchar(100) NOT NULL,
```

Comment:
Changed to `country_code` for simplified standardization vs. storing the full country string.